BSP/buttons/buttons.h

```diff
@@ -0,0 +1,30 @@
+#ifndef BUTTONS_H
+#define BUTTONS_H
+
+#include <stdbool.h>
+
+typedef enum button{
+    BUTTON_CENTER,
```

Comment:
I don't know if this is guaranteed to be a 0 otherwise.
```suggestion
    BUTTON_CENTER = 0,
```

Reply:
I checked it before committing https://stackoverflow.com/questions/6434105/are-default-enum-values-in-c-the-same-for-all-compilers/30312219